DelvCD/Config/StatusTrigger.cs

```diff
@@ -226,5 +239,21 @@ private void AddTriggerData(TriggerData triggerData)
             _triggerNameInput = TriggerName;
             TriggerData.Add(triggerData);
         }
+
+        private Boolean DoesActorMatchTriggerSourceType(GameObject actor, TriggerSourceType sourceType) {
+            // Any basically means we don't care about checking the type, so just return true
+            if (sourceType is TriggerSourceType.Any) { return true; }
+
+            // Sanity check
+            if (actor == null || actor is not BattleChara) { return false; }
+
+            // Do we want to include BattleNpcSubKind.Pet or BattleNpcSubKind.Chocobo as friendly?
+            if (sourceType is TriggerSourceType.Friendly && actor is PlayerCharacter) { return true; }
+
+            if (sourceType is TriggerSourceType.Enemy && actor is BattleNpc && ((BattleNpc)actor).BattleNpcKind is BattleNpcSubKind.Enemy) { return true; }
```

Comment:
Also, I didn't actually test this code, please do that before pushing :P

Reply:
I made the changes requested, though there is an issue with enemy-only triggers that I will be getting around to debugging and fixing tonight.

Reply:
Alright, I've worked through it and tested it. I've added some comments in the `DoesActorMatchTriggerSourceType` private method for a bit of clarity in case anyone needs to touch that again. Let me know if there is anything else you'd like me to change.